### DISCUSSION TITLE
[feature request] reference field

### DISCUSSION THREAD

**Discussion by trailjeep** [knqyf263/pet#274] [Ideas]
First, thanks for pet. My semi-functional brain relies on it!

It would be nice to have an additional reference field to place links to additional info such as where I found the particular command / or thread with additional discussion.
Currently, I am adding into the description field, but it would be much cleaner with a dedicated reference field.

Thanks

[Edit]
I have found a nice workaround (`\n` newline & `\t` tab work fine in the description field) so I add to Description for example:
`"BASH script pass command options as array\n\t[Using shell variables for command options](https://unix.stackexchange.com/a/459369)"`

**Comment by RamiAwar** [comment]
Hey @trailjeep ! I like the idea as well, I think it can be nice. 

How would you imagine using this however? I can imagine it doesn't make sense to show this field's contents in the search window, cause this sounds more like a long term reference.

Maybe in the pet edit command you could see this stuff? Then you can fill it in in a similar way to how we fill in tags (pet new -t).
